src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets

```
@@ -62,8 +62,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <WasmFingerprintDotnetJs Condition="'$(_TargetingNET80OrLater)' == 'true' and '$(_TargetingNETBefore100)' == 'true'">false</WasmFingerprintDotnetJs>
     <WasmEnableWebcil Condition="'$(WasmEnableWebcil)' == '' and ('$(TargetFrameworkIdentifier)' != '.NETCoreApp' or '$(_TargetingNET80OrLater)' != 'true')">false</WasmEnableWebcil>
 
-    <!-- Turn off parts of the build that do not apply to Blazor projects -->
-    <GenerateRuntimeConfigurationFiles>false</GenerateRuntimeConfigurationFiles>
+    <!-- Turn off parts of the build that do not apply to Blazor projects until Net10 -->
+    <GenerateRuntimeConfigurationFiles Condition="'$(GenerateRuntimeConfigurationFiles)' == '' and '$(_TargetingNETBefore100)' == 'true'">false</GenerateRuntimeConfigurationFiles>
```

```suggestion
    <GenerateRuntimeConfigurationFiles Condition="''$(_TargetingNETBefore100)' == 'true'">false</GenerateRuntimeConfigurationFiles>
```

I think we should maintain the behavior from previous SDK when targeting downlevel versions